src/core/connection.ts

```diff
@@ -87,7 +91,26 @@ export class Connection extends EventEmitter {
     if (!this.options.redis) {
       await this.redis.select(this.options.database);
     }
-    await connectionTest();
+    await connectionTestAndLoadLua();
```

Comment:
while trying o make proof of concept for https://github.com/actionhero/node-resque/pull/475#discussion_r524590731

i noticed that `connectionTestAndLoadLua` is being called twice for code branch if `this.options.redis` is true-ish:
1. immediately after assigning this.redis (line 62)
2. prior returning from the function (line 94)

Reply:
Good catch!   I think that removing line 62 is probably the right choice